src/core/components/tabs/tabsList/tabsList.tsx

```diff
@@ -11,6 +11,14 @@ export const TabsList: React.FC<ITabsListProps> = (props) => {
 
     const tabsListClassNames = classNames('flex gap-x-6', { 'border-b border-neutral-100': isUnderlined }, className);
 
+    /* If there is only a single child then the tabs are redundant and we just show the content
+    In this case we don't render the tabs list. Note that we should always set either a defaultValue or value
+    prop on the Tabs.Root component if there is a possibility of only a single child being present
+    Otherwise no tab will be selected and no content will render */
```

Comment:
I would avoid writing the same doc twice, here and on the story, we can keep a short comment here
```suggestion
    // If there is only a single child then the tabs are redundant and we just show the content
```